test/mpi/threads/pt2pt/commperthread_perf.c

```diff
@@ -0,0 +1,240 @@
+/*
+ * Copyright (C) by Argonne National Laboratory
+ *     See COPYRIGHT in top-level directory
+ */
+
+/* This program provides a simple test of send-receive performance between
+   two (or more) processes.  This sometimes called head-to-head or
+   ping-ping test, as both processes send at the same time.
+*/
+
+#include "mpi.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include "mpitest.h"
+#include "mpithreadtest.h"
+
+#define NUM_THREADS 4
```

Comment:
Can we make it an argument, for example `-thread=8`? The default 4 works.

Reply:
Added an optional argument in the new version. Kept the argument name as `-nthreads`.